docs/tutorials/platform_beginner/2_create_the_content_model.md

```diff
@@ -0,0 +1,64 @@
+
+To get full information, read the [content model](../guide.content_model.md) documentation page.
+Below is a short, straight-to-the-point introduction that covers point essential for this tutorial.
+
+## Content model overview
+
+The eZ Platform content Repository is centered around **Content items**. A Content item is a single piece of content, for example an article, a product review, a place, etc.
+
+Every Content item is an instance of a **Content Type**. Content Types define what **Fields** are included in each Content item.
+For example, an article could include a Fields such as *title*, *image*, *abstract*, *article's body*, *publication date* and *list of authors*.
+
+Fields can belong to one of the installed **Field Types**, about 30 in the default distribution.
+Each Field Type is built to represent a specific type of data: a text line, a block of rich text, an image, a collection of relations to Content items, etc.
+You can find a complete list in the [Field Types reference](../../api/field_type_reference.md) section.
+Every Field Type may have its own options, and comes with its own editing and viewing interfaces.
+
+## Create a Content Type
+
+The site will use two Content Types: **Ride** and **Landmark**.
+A Ride is a route of a bike trip. It can include one or more Landmarks - interesting places you can see along the way.
+More than one Ride can visit the same Landmark, so in database terms this is an N-N relationship model.
+
+In this step you'll create the first Content Type, Ride.
+
+Go to the admin interface (`<yourdomain>/ez`) and log in with the default username: `admin` and the default password: `publish`. 
+
+Go to the **Admin Panel** in the top bar and choose **Content types** in the sub menu:
+
+![Navigation hub](img/bike_tutorial_nav_hub.png)
+
+You will see a list of **Content Type Groups**. They are used to group Content Types in a logical way.
+
+Select **Content** and then **Create a content type**. 
+
+![Create a content type button](img/bike_tutorial_create_content_type.png)
+
+Fill the form with this basic info: 
+
+- **Name**: Ride
+- **Identifier**: `ride`
+
+Then create all Fields with the following information: 
+
+| Field Type   | Name             | Identifier       |  Required | Searchable | Translatable |
+| ------------ | ---------------- | ---------------- | --------- | ---------- | ------------ |
+| Text line    | Name             | `name`           | yes       | yes        | yes          |
+| Image        | Photo            | `photo`          | no        | no         | no           |
+| Rich Text    | Description      | `description`    | yes       | yes        | yes          |
+| Map location | Starting point   | `starting_point` | yes       | yes        | no           |
+| Map location | Ending point     | `ending_point`   | yes       | yes        | no           |
+| Integer      | Length           | `length`         | yes       | yes        | no           |
+
```

Comment:
I wasn't sure which button should I click to confirm: OK or Apply. It could be mentioned here.